packages/apollo/src/commands/client/__tests__/generate.test.ts

```diff
@@ -191,6 +191,26 @@ describe("client:codegen", () => {
       expect(fs.readFileSync("API.ts").toString()).toMatchSnapshot();
     });
 
+  test
+    .fs({
+      "schema.json": fullSchemaJsonString,
+      "queryOne.graphql": simpleQuery.toString(),
+      "my.config.js": defaultConfig
+    })
+    .command([
+      "client:codegen",
+      "--config=my.config.js",
+      "--target=typescript",
+      "--fileExtension=.d.ts",
```

Comment:
Can we change this flag to be only what's after the dot? l

Like `d.ts` instead of `.d.ts`? I feel like only the part after the dot is technically the file extension. Maybe I'm wrong though? What do you think?